lib/Controller/ApiController.php

```diff
@@ -110,6 +117,25 @@ public function info() {
 
 	}
 
+	/**
+	 * @return DataResponse
+	 */
+	public function BasicData(): DataResponse {
+		$servertime = $this->Os->getTime();
+		$uptime = $this->Os->getUptime();
+		$timeservers = $this->Os->getTimeServers()[0];
+		return new DataResponse(array('servertime'=>$servertime,'uptime'=>$uptime,'timeservers'=>$timeservers));
```

Comment:
use `[]` instead of `array()` and spaces around `=>`. Also spaces after `,`. Best would be to put every entry on a new line.